.npmignore

```diff
@@ -1,34 +0,0 @@
-# compiled output
```

Comment:
Because we have [this](https://github.com/ember-cli/ember-cli-htmlbars/blob/master/package.json#L21-L23).